code/modules/hydroponics/beekeeping/beebox.dm

```diff
@@ -7,23 +7,32 @@
 
 
 
-/mob/proc/bee_friendly()
-	return 0
+/mob/proc/bee_friendly() //Because we, are, your friends
+	return FALSE //You'll, never be(e) alone again    //NB, returning 0 here means that the bees are actually not your friends (they lied to you)
```

Comment:
```suggestion
	return FALSE //You'll, never be(e) alone again
```

No alignment indentation (the tabs before the //NB). Also, your NB comment is useless--the proc is named `bee_friendly`, this should be obvious.

Reply:
but the comments around this are kind of misleading without the NB one